addons/transitions/Transitions.gd

```diff
@@ -150,16 +148,21 @@ func _common_post_fade(data:Array, new_scene) -> void:
 	if fade_scene in _root:
 		_root.remove_child(fade_scene)
 
-func _set_scene(new_scene):
+func _set_scene(new_scene:Node):
```

Comment:
Setting this to `Node` will not allow me to pass a scene where the root is `Control` (which is most of my UI scenes). Can you please remove this?

Reply:
Control inherits node so there should be no problem with that, in fact everything inherits node